.github/workflows/build-publish-develop.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Build Chainlink and push image with latest develop tag'
```

Comment:
How do you feel about a less verbose name? It's easy to overflow actions names where its hard to see what action is running. EG.

`Push develop to private ECR`